macOS/DuckDuckGo/Permissions/Model/PermissionManager.swift

```diff
@@ -26,35 +26,25 @@ protocol PermissionManagerProtocol: AnyObject {
     typealias PublishedPermission = (domain: String, permissionType: PermissionType, decision: PersistedPermissionDecision)
     var permissionPublisher: AnyPublisher<PublishedPermission, Never> { get }
 
+    func hasPermissionPersisted(forDomain domain: String, permissionType: PermissionType) -> Bool
```

Comment:
This (and `persistedPermissionTypes`) are added to the protocol because they were accessed from a PermissionManager instance in some places.